.travis.yml

```diff
@@ -208,16 +212,16 @@ script:
     Description = ClickHouse DSN (ANSI, localhost)
     Url         = http://${CLICKHOUSE_SERVER_IP}
     Timeout     = 30
-    Trace       = yes
-    TraceFile   = /tmp/clickhouse-odbc-driver-trace.log
+    Log         = yes
```

Comment:
Maybe something less general? Like DebugLog?

Reply:
Technically, it enables/redirects any driver-specific log. Verbosity, when and if implemented, may only control it further.

Reply:
Renamed to `DriverLog` and `DriverLogFile`.